zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -306,71 +236,45 @@ impl<F: Field> Term<F> {
     }
 }
 
-pub(crate) struct AssignedECDSA<F: Field> {
-    pk_x_le: [AssignedValue<F>; 32],
-    pk_y_le: [AssignedValue<F>; 32],
-    msg_hash_le: [AssignedValue<F>; 32],
+pub(crate) struct AssignedECDSA<F: Field, FC: FieldChip<F>> {
+    pk: EccPoint<F, FC::FieldPoint>,
+    msg_hash: OverflowInteger<F>,
+    sig_is_valid: halo2_base::AssignedValue<F>,
 }
 
 #[derive(Debug)]
 pub(crate) struct AssignedSignatureVerify<F: Field> {
-    pub(crate) address: AssignedValue<F>,
+    pub(crate) address: halo2_base::AssignedValue<F>,
```

Comment:
I think we need to include `sig_is_valid` in this struct s.t. tx circuit can assert it's always true.